autorest/codegen/templates/sample.py.jinja2

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+{{ code_model.options['license_header'] }}
+
+{{ imports }}
+"""
+The sample just shows how to use the method and may not run successfully.
+# PREREQUISITES
+{% if "credential" in client_params %}
+    pip install azure-identity
+{% endif %}
+    pip install {{ (code_model.options["package_name"] or code_model.client.name)|lower }}
```

Comment:
is the plan to add async as well?

Reply:
we could generate sync sample first and then generate async in the future. (keep rhythm with js/java)